dc.py

```diff
@@ -541,19 +543,24 @@ def run(self):
 
         host = self.ui.hostName.text()
         port = self.ui.basePortSpin.value()
+        self.settings.setValue("host",host)
+        self.settings.setValue("port",int(port))
         return (host, port)
 
 
 def main():
-    app = QtWidgets.QApplication(sys.argv)
-    host, port = "localhost", 5500
+    settings = QSettings("qsg","dastard-commander")
```

Comment:
In `ndfb_server/NDFBServerGUI.cpp` we have

```c++
    QCoreApplication::setOrganizationName("NIST Quantum Sensors");
    QCoreApplication::setApplicationName("NDFB_Server");
```

Maybe you could use the same organization name? Not that it really matters...